webpack_config/webpack.config.base.js

```diff
@@ -33,7 +45,11 @@ var config = {
             {
                 test: /fg-loadcss\/src\/onloadCSS/,
                 loader: 'exports?onloadCSS'
-            }
+            },
+            {
+                test: /\.html$/,
+                loader: 'vue-html'
```

Comment:
Does this loader conflict with the Django template syntax at all? I can't figure out what exactly it's supposed to do.